unittests/src/main/java/uk/ac/manchester/tornado/unittests/common/TornadoTestBase.java

```diff
@@ -54,7 +54,10 @@ public void before() {
             }
         }
 
-        if (!wasDeviceInspected) {
+        /* Virtual devices support a single device. Therefore, it doesn't make sense to change the device even if
```

Comment:
Update the comment:

`Virtual Device execution assumes an environment with a single device. Therefore, there is no need to change.`